wiki/Article_styling_criteria/en.md

```diff
@@ -0,0 +1,23 @@
+---
+tags:
+  - wiki standards
+---
+
+# Article styling criteria
+
+*For news posts, see: [News Styling Criteria](/wiki/News_Styling_Criteria)*
+
+The Article Styling Criteria (ASC) serve as the osu! wiki's enforced styling standards to keep consistency in clarity, formatting, and layout in all articles.
+
+All articles must have having proper grammar, correct spelling, and correct information.
```

Comment:
This seems a bit above the line. I like the old lenient wording better, as we do make mistakes and wikis are always WIP (i.e. people are always editing them).

> All articles should aim at having proper grammar, correct spelling, and correct information.

```suggestion
All articles should aim at having proper grammar, correct spelling, and correct information.
```

Or if not:

```suggestion
All articles must have proper grammar, correct spelling, and correct information.
```